APIBacklog/documentation/SupportingDocuments/API proposals/APIproposal_BlockchainPublicAddress_Telefonica.md

```diff
@@ -0,0 +1,12 @@
+| **Field** | Description | 
+| ---- | ----- |
+| API family name | Blockchain Public Address |
+| API family owner| Telefonica  |
+| API summary | **High level description**<br> This API allows to manage a Blockchain Public Address associated to a phone number, i.e. to retrieve the blockchain public address(es) and to bind/unbind a Blockchain Public Address. The Block Public Address is referred to as Decentralized IDentifier (DID). With the proposed API, telco service providers have the opportunity to provide 3rd parties with the following capability: pairing phone number with DID whenever this 3rd party wants to offer its customers a way to make transactions more easily based on the phone number instead of the DID. <br><br>**Rationale**<br>DID is composed of 26-35 alphanumeric characters in length. This identifier is not memorisable or usable by a person, so there is the possibility of positioning the telephone number as the predominant identifier on web3. This new identity protocol will make it possible to have a system similar to the Telcos' bizum in web3 ecosystem.
```

Comment:
"DID is composed of 26-35 alphanumeric characters" 

DIDs have no length restriction (as far as I know); W3C ABNF
did                = "did:" method-name ":" method-specific-id
method-name        = 1*method-char
method-char        = %x61-7A / DIGIT
method-specific-id = *( *idchar ":" ) 1*idchar
idchar             = ALPHA / DIGIT / "." / "-" / "_" / pct-encoded
pct-encoded        = "%" HEXDIG HEXDIG

However, a bitcoin address is indeed a 26-35 alphanumeric character identifier (while a wallet address in Ethereum is a 42-character hexadecimal address). Should this say 'wallet address' instead of DID?

Reply:
Text is updated in this commit: https://github.com/camaraproject/WorkingGroups/pull/154/commits/6bec4f9f8a0c0a35295992f9c57c2c4ced689ad5
Please check, we reworded it to make sentence clearer by using Bitcoin length just as an example